### DISCUSSION TITLE
Can I disconnect the raspberry pi from electricity?

### DISCUSSION THREAD

**Discussion by xoaugustas** [AdguardTeam/AdGuardHome#5232] [Q&A]
I'm curious, do I need to keep the pi on always to make adguard server work?
Or I just power it on whenever I need to change settings?
Any help?

**Comment by ppfeufer** [comment]
Well, when you power down the Pi, AGH is no longer available. Such is the nature of a device that needs electricity to work ... it and all the software on it only work when powered on ...
